frontend/packages/ceph-storage-plugin/src/components/create-storage-system/external-storage/ibm-flashsystem/index.tsx

```diff
@@ -0,0 +1,162 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Form, FormGroup, TextInput, Button, ValidatedOptions } from '@patternfly/react-core';
+import { EyeSlashIcon, EyeIcon } from '@patternfly/react-icons';
+import { SecretKind, apiVersionForModel } from '@console/internal/module/k8s';
+import { SecretModel } from '@console/internal/models';
+import { isValidUrl } from '@console/shared';
+import { FlashsystemState, IBMFlashsystemKind } from './type';
+import { IBMFlashsystemModel } from './models';
+import { CreatePayload, ExternalComponentProps, CanGoToNextStep } from '../types';
+
+export const FlashsystemConnectionDetails: React.FC<ExternalComponentProps<FlashsystemState>> = ({
+  setFormState,
+  formState,
+}) => {
+  const { t } = useTranslation();
+  const [reveal, setReveal] = React.useState(false);
+  const [endpointValid, setEndpointValid] = React.useState(ValidatedOptions.default);
+
+  const onChange = (value: string) => {
+    setFormState('endpoint', value);
+    value && isValidUrl(value)
+      ? setEndpointValid(ValidatedOptions.success)
+      : setEndpointValid(ValidatedOptions.error);
+  };
+
+  return (
+    <Form>
+      <FormGroup
+        label={t('ceph-storage-plugin~Endpoint')}
+        fieldId="endpoint-input"
+        isRequired
+        validated={endpointValid}
+        helperText={t('ceph-storage-plugin~Rest API IP address of IBM Storage FlashSystem.')}
```

Comment:
I think it should be :
```suggestion
        helperText={t('ceph-storage-plugin~Rest API IP address of IBM FlashSystem Storage.')} 
```
??